.github/workflows/nightly.yml

```diff
@@ -0,0 +1,42 @@
+name: Nightly tests
+
+on:
+    workflow_dispatch: # note: if started manually, it won't run all matrix
```

Comment:
Can we add the option to choose full or not?

Reply:
It did not work for me. I'll try to fix it and restore.